app/serializers/profile_serializer.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class ProfileSerializer < ActiveModel::Serializer
+  include Rails.application.routes.url_helpers
```

Comment:
I don't think this line `include Rails.application.routes.url_helpers` is required

Reply:
good point, I think we don't need it in PersonSerializer too.